CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 0.7.6-dev
  - detect and track redirects in `GooseRawRequest`
+ - redirect of test base_url are sticky and affect subsequent requests
```

Comment:
Redirects are sticky OR redirect is sticky

Reply:
thanks, grammar now fixed